src/components/DayPicker.jsx

```diff
@@ -104,14 +104,63 @@ export default class DayPicker extends React.Component {
   }
 
   getMonthHeightByIndex(i) {
-    const $transitionContainer = $(ReactDOM.findDOMNode(this.refs.transitionContainer));
-    return this.getMonthHeight($transitionContainer.find('.CalendarMonth').eq(i));
+    return this.getMonthHeight(
+      ReactDOM.findDOMNode(this.refs.transitionContainer).querySelectorAll('.CalendarMonth')[i]
+    );
   }
 
-  getMonthHeight($el) {
+  getMonthHeight(el) {
+    const caption = el.querySelector('.js-CalendarMonth__caption');
+    const grid = el.querySelector('.js-CalendarMonth__grid');
+
     // Need to separate out table children for FF
-    return $el.find('.js-CalendarMonth__caption').outerHeight(true) +
-      $el.find('.js-CalendarMonth__grid').height() + 1; // for border
+    return (
+      this.calculateDimension(caption, 'height', 'outer', true) +
+      this.calculateDimension(grid, 'height') +
+      1 // for border
```

Comment:
Comments should go before lines. https://github.com/airbnb/javascript#comments--singleline